google/cloud/bigtable/data/_async/client.py

```diff
@@ -132,6 +132,10 @@ def __init__(
         client_options = cast(
             Optional[client_options_lib.ClientOptions], client_options
         )
+        self._emulator_host = os.getenv(BIGTABLE_EMULATOR)
+        if self._emulator_host is not None and credentials is None:
+            # use insecure channel if emulator is set
+            credentials = google.auth.credentials.AnonymousCredentials()
```

Comment:
How are the credentials related to adding conformance tests?

Reply:
Because this is the first time they are being run in a CI environment without GCP authentication, which was raising an error even in emulator mode

Reply:
Looks good to me. It seems like this situation is a lot like something I had to do in datastore for setting insecure credentials when the emulator is being used. See https://github.com/googleapis/nodejs-datastore/pull/1164/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R516. I guess this is a bug fix as well then?

I might recommend in a future PR to also use the insecure credentials if an endpoint is provided in the constructor to hit the emulator when this environment variable is not provided.